synapse/util/logcontext.py

```diff
@@ -42,11 +55,21 @@ def __str__(self):
         def copy_to(self, record):
             pass
 
+        def start(self):
```

Comment:
So that I don't need to check if the context is the sentinel when using the functions.